packages/react-components/react-card/README.md

```diff
@@ -1,13 +1,85 @@
 # @fluentui/react-card [ALPHA]
 
-**React Card components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+**React Card components for [Fluent UI React](https://react.fluentui.dev)**
 
 **⚠️ Please note that functionality is still being added to this package. Due to lockstep versioning, the version of this package is aligned with the others in _react-components_.**
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+These are not production-ready components and **should never be used in a product**. This space is useful for testing new components whose APIs might change before final release.
+
+![example of a Card component in usage](./assets/card-example.png)
+
+## Usage
 
 To import React Card components:
 
 ```js
-import { ComponentName } from '@fluentui/react-card';
+import { Card, CardPreview, CardHeader, CardFooter } from "@fluentui/react-components/unstable';
+```
+
+Example Card usage, shown above:
+
+<!-- prettier-ignore-start -->
```

Comment:
why is this needed ?

Reply:
Was there because of how the formatting would look when leveraging our composition model. Then I changed it to an element and forgot about it 😶‍🌫️